region/wreckedship/main/Wrecked Ship Energy Tank Room.json

```diff
@@ -380,17 +384,22 @@
       "link": [2, 1],
       "name": "Ceiling Bomb Jump",
       "requires": [
-        "h_canCeilingBombJump"
+        "h_canCeilingBombJump",
+        "canLongCeilingBombJump",
+        "canBePatient"
       ]
     },
     {
       "link": [2, 1],
       "name": "G-mode Morph Ceiling Bomb Jump",
       "requires": [
         "h_canArtificialMorphCeilingBombJump",
+        "canLongCeilingBombJump",
+        "canBeVeryPatient",
         {"obstaclesCleared": ["A"]}
       ],
-      "note": "Touch the item while remaining in artificial morph. Ceiling bomb jump back to the right, then use x-ray to cancel g-mode and obtain the item."
+      "note": "Touch the item while remaining in artificial morph. Ceiling bomb jump back to the right, then use x-ray to cancel g-mode and obtain the item.",
+      "devNote": "This strat alone would only require canBePatient, but it is only possible after Ceiling Bomb Jumping there, so it would be a combined 4 minues."
```

Comment:
```suggestion
      "devNote": "This strat alone would only require canBePatient, but it is only possible after Ceiling Bomb Jumping there, so it would be a combined 4 minutes."
```